res/css/views/settings/_LayoutSwitcher.pcss

```diff
@@ -15,79 +15,76 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-.mx_LayoutSwitcher_RadioButtons {
+.mx_LayoutSwitcher_LayoutSelector {
     display: flex;
-    flex-direction: row;
-    gap: 24px;
-    width: 100%;
+    flex-direction: column;
+    gap: var(--cpd-space-4x) !important;
```

Comment:
What is this overriding?

Reply:
The css settings has a default gap of `10px` for the form. Overriding it to have to have the same gap than in the figma design. I can had a comment to explain this.

Reply:
Yes please